appmgr/src/net/wifi.rs

```diff
@@ -213,31 +212,31 @@ fn display_wifi_info(info: WiFiInfo, matches: &ArgMatches<'_>) {
         &info
             .connected
             .as_ref()
-            .map_or("[N/A]".to_owned(), |c| format!("{}", c)),
+            .map_or("[N/A]".to_owned(), |c| format!("{}", c.0)),
         &info
-            .signal_strength
+            .connected
             .as_ref()
-            .map_or("[N/A]".to_owned(), |ss| format!("{}", ss)),
+            .and_then(|x| info.ssids.get(x))
+            .map_or("[N/A]".to_owned(), |ss| format!("{}", ss.0)),
         &format!("{}", info.country.alpha2()),
         &format!("{}", info.ethernet)
     ]);
     table_global.print_tty(false);
 
     let mut table_ssids = Table::new();
-    table_ssids.add_row(row![bc => "SSID",]);
-    for ssid in &info.ssids {
-        let mut row = row![ssid];
-        if Some(ssid) == info.connected.as_ref() {
-            row.iter_mut()
-                .map(|c| {
-                    c.style(Attr::ForegroundColor(match &info.signal_strength {
-                        Some(100) => color::GREEN,
-                        Some(0) => color::RED,
-                        _ => color::YELLOW,
-                    }))
-                })
-                .collect::<()>()
-        }
+    table_ssids.add_row(row![bc => "SSID", "STRENGTH"]);
+    for (ssid, signal_strength) in &info.ssids {
+        let mut row = row![&ssid.0, format!("{}", signal_strength.0)];
+        row.iter_mut()
+            .map(|c| {
+                c.style(Attr::ForegroundColor(match &signal_strength.0 {
```

Comment:
Match something like
![image](https://user-images.githubusercontent.com/2364004/147841416-09955051-8026-48e6-acba-51054009bffa.png)
Also, all the ssid's would not share the same strength so the previous implementation was wrong.

Reply:
Now this will look like 
![image](https://user-images.githubusercontent.com/2364004/147841469-d23808a0-0d39-4d0e-82e1-dffa69c19823.png)

Reply:
Maybe all the networks should be like the frontend?
![image](https://user-images.githubusercontent.com/2364004/147841560-0b75f5e7-5835-4da3-a449-67210f7ddfc5.png)